pkg/resources/grant_privileges_to_role.go

```diff
@@ -65,6 +65,7 @@ var grantPrivilegesToRoleSchema = map[string]*schema.Schema{
 						"INTEGRATION",
 						"FAILOVER GROUP",
 						"REPLICATION GROUP",
+						"EXTERNAL VOLUME",
```

Comment:
You did not change the description. And they are also listed there. You will also have to regenerate docs after changing the description.